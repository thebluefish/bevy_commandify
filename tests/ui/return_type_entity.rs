use bevy_commandify::*;

#[entity_command]
fn foo(world: &mut World) -> Command { }

fn main() { }