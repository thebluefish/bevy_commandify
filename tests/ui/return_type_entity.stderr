error: command may not define a return type, except for `&mut Self`
 --> tests/ui/return_type_entity.rs:4:30
  |
4 | fn foo(world: &mut World) -> Command { }
  |                              ^^^^^^^
